docs/_collections/_notify-repos/nhs-notify-web-platform.md

```diff
@@ -0,0 +1,13 @@
+---
+repo-name: nhs-notify-web-platform
+owners: ["rossbugginsnhs"]
+name: Notify Web Platform
+description: Notify Micro frontend web orchestration platform
+layout: nhs-notify-web-platform
```

Comment:
```suggestion
layout: notify-repo
```

Reply:
ta for that, good spot. Have committed that but it now wants re approval from you :-) cheers